contracts/VToken.sol

```diff
@@ -765,12 +979,14 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
      * @param payer the account paying off the borrow
      * @param borrower the account with the debt being payed off
      * @param repayAmount the amount of underlying tokens being returned, or -1 for the full outstanding amount
+     * @param interestRateMode The interest rate mode at of the debt the user wants to repay: 1 for Stable, 2 for Variable
```

Comment:
```suggestion
     * @param interestRateMode The interest rate mode of the debt the user wants to repay: 1 for Stable, 2 for Variable
```

This is because a user could have a stable and variable position?

Reply:
Yes, 
Fixed, can check the changes at [this commit](2981e0d08b247b7c7a7dc3461e9701d7d593c520).